server/komodo-rest/src/test/java/org/komodo/rest/relational/AbstractKomodoServiceTest.java

```diff
@@ -238,10 +234,16 @@ protected void loadVdbs() throws Exception {
         Assert.assertEquals(4, _restApp.getVdbs(USER_NAME).length);
     }
 
-    protected void loadDataServices() throws Exception {
+    protected void loadStatesDataService() throws Exception {
         _restApp.importDataservice(TestUtilities.usStatesDataserviceExample(), USER_NAME);
     }
```

Comment:
Was expecting there to be more test data services available over time hence the generic method name.